express-api/eslint.config.mjs

```diff
@@ -0,0 +1,32 @@
+import typescriptEslint from '@typescript-eslint/eslint-plugin';
+import typescriptParser from '@typescript-eslint/parser';
+import prettierPlugin from 'eslint-plugin-prettier';
+
+export default {
+  // Specify the parser for TypeScript
+  languageOptions: {
+    parser: typescriptParser,
+  },
+  plugins: {
+    '@typescript-eslint': typescriptEslint,
+    prettier: prettierPlugin,
+  },
+  // Define your ESLint rules
+  rules: {
+    'prettier/prettier': 'warn',
+    'no-extra-boolean-cast': 'off',
+    'no-unsafe-optional-chaining': 'off',
+    'no-prototype-builtins': 'off',
+    'no-console': 'error',
+    '@typescript-eslint/ban-ts-comment': 'off',
+    '@typescript-eslint/no-inferrable-types': 'off',
+    '@typescript-eslint/no-empty-function': 'off',
+    '@typescript-eslint/no-non-null-assertion': 'off',
+    '@typescript-eslint/no-explicit-any': 'warn',
+    '@typescript-eslint/no-duplicate-enum-values': 'error',
+  },
+  // Ignore specific files and directories
+  ignores: ['node_modules/', 'package-lock.json', 'dist/', 'coverage/'],
```

Comment:
I seem to get this warning from the coverage folder still. Not sure why it's not picking it up.
<img width="638" alt="image" src="https://github.com/user-attachments/assets/5d3f444a-d214-4dbb-bc40-25bf92df16e9">

Reply:
> I seem to get this warning from the coverage folder still. Not sure why it's not picking it up. <img alt="image" width="638" src="https://private-user-images.githubusercontent.com/37922247/362414460-5d3f444a-d214-4dbb-bc40-25bf92df16e9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQ4NzE3NzMsIm5iZiI6MTcyNDg3MTQ3MywicGF0aCI6Ii8zNzkyMjI0Ny8zNjI0MTQ0NjAtNWQzZjQ0NGEtZDIxNC00ZGJiLWJjNDAtMjViZjkyZGYxNmU5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODI4VDE4NTc1M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWNjMTBlMDE4MjM3ZTI5NGY0MDkwNGRiOTI0NzRlMjI2MDdmOTc3ZjU3MjgxZjdjNmZlYTk3ZTNkMjUxYmQ1NWImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.kdQIivU-qOIGeV-Irifk4jpbYO50VgSAXxf9J7Cl0Q0">

I removed the eslint-disable from sorter and block-navigation files in local environment, but it is not picked up as changes when I commit.

Reply:
Those files are autogenerated when you run tests and they shouldn't be committed. The config should be telling eslint to ignore files in those paths though.